### DISCUSSION TITLE
How to use test telegram environment?

### DISCUSSION THREAD

**Discussion by rzxczxc** [telegram-bot-sdk/telegram-bot-sdk#107] [Q&A]
Hi! How I can use telegram [test environment ](https://core.telegram.org/bots/webapps#using-bots-in-the-test-environment)with this package or with v3 (https://github.com/irazasyed/telegram-bot-sdk)

Telegram test env uses `/test/` before method name in URL paths

**Comment by adshodgson** [comment]
Any update here?

@kreexzoi one workaround i used for the time being was to modify the vendor prepareRequest function temporarily.

```
public function prepareRequest(TelegramRequest $request): array
    {
        $url = $this->baseBotUrl.$request->getAccessToken().'/test/'.$request->getEndpoint();

        return [$url, $request->getMethod(), $request->getHeaders(), $request->isAsyncRequest()];
    }
```
